src/gol.js

```diff
@@ -0,0 +1,40 @@
+export function countLiveNeighbours(board, i, j) {
```

Comment:
I like the order of functions. it is good practice to define functions before you use them.